src/univalue/test/unitester.cpp

```diff
@@ -157,6 +157,8 @@ int main (int argc, char *argv[])
 
     unescape_unicode_test();
 
-    return 0;
+    char buf[] = "___[1,2,3]___";
+    UniValue val;
+    return val.read(buf + 3, 7) ? 0 : 1;
```

Comment:
nit: Could use `assert` like in the rest of this file?

`assert(val.read(buf + 3, 7))`

Reply:
Done